src/components/Layout/Layout.tsx

```diff
@@ -88,7 +88,7 @@ export const Layout = ({
 
   const showNextPrev = NEXT_PREVIOUS_SECTIONS.some(
     (section) =>
-      asPathWithNoHash.includes(section) &&
+      pathname.includes(section) &&
```

Comment:
Using pathname so that we can check for gen1 paths without needing to know the exact platform. `pathname` will look like this: `/gen1/[platform]/my/page` or `/[platform]/my/page/here`